src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -298,6 +304,23 @@ def notify_issue_update(testcase, status):
     oss_fuzz_github.close_issue(testcase)
 
 
+def check_miracleptr_status(testcase):
+  """Look for MiraclePtr status string and return the appropriate label."""
+  label = None
+  stacktrace = data_handler.get_stacktrace(testcase)
+  for line in stacktrace.split('\n'):
+    if CHROMIUM_MIRACLEPTR_PROTECTED_REGEX.match(line):
```

Comment:
per above, just have one regex, and map the captured group tro the label.

Reply:
Done.